src/java.base/share/classes/jdk/internal/classfile/impl/StackCounter.java

```diff
@@ -343,6 +335,51 @@ public StackCounter(LabelContext labelContext,
         maxStack += rets * maxStack;
     }
 
+    private static int countMethodStack(Utf8Entry descriptor, boolean subReturn) {
```

Comment:
Since we use this for both locals and stack, perhaps "countMethodSlots" would be better? And maybe the "subReturn" could be a flag to switch between locals/stack

Reply:
I've changes the name to the suggested "countMethodSlots" and "subReturn" to "asStackDelta" , thanks.